README.md

```diff
@@ -52,39 +47,55 @@ import UAPusher
 try drop.addProvider(UAPusher.Provider.self)
 ```
 
-### Example
-
+### Simple example
 ```swift
 let body = try JSON(node: [
-	"audience": "all",
-	"device_types": [
-		"ios",
-		"android"
-	],
-	"notification": [
-		"alert": "hello world"
-	]
+    "audience": "all",
+    "device_types": [
+        "ios"
+    ],
+    "notification": [
+        "alert": "hello world"
+    ]
 ])
         
 let request = UARequest(body: body)
+
 do {
-	let response = try drop.uapusher?.send(request: request)
-	if response.status == .accepted {
-		print("Push send..")
-	}
+    let response = try drop.uapusher?.send(request: request)
+    if response.status == .accepted {
+        print("Push send..")
```

Comment:
Would it make more sense to word it "Push sent.." ?

Reply:
It certainly would :D